### DISCUSSION TITLE
I need help to access the Android API with jclass

### DISCUSSION THREAD

**Discussion by Razer-0** [beeware/toga#2499] [Debugging help]
Hello, I need help to access the Android API with jclass. Currently, when trying to obtain the getApplicationContext I get the following error

`E/AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.phone.callwave/org.beeware.android.MainActivity}: com.chaquo.python.PyException: NotImplementedError: android.content.Context.getContentResolver is abstract and cannot be called`

```
from java import jclass

# Obtener la clase Context


Context = jclass('android.content.Context')

# Obtener el contexto de la aplicación
context = Context.getApplicationContext()

```

**Comment by Razer-0** [comment]
I don't know what I'm doing wrong or overlooking... I would appreciate any help, thank you.

**Reply by mhsmith** [reply]
You can access the `Activity` (which is a subclass of `Context`) as `app._impl.native`, where `app` is your `toga.App` object.
